addons/l10n_in/models/account_invoice.py

```diff
@@ -542,6 +542,14 @@ def _get_name_invoice_report(self):
 
     def _post(self, soft=True):
         """Use journal type to define document type because not miss state in any entry including POS entry"""
+        for move in self:
+            chart_template = move.company_id.chart_template
+            fiscal_country = move.company_id.account_fiscal_country_id
+            if chart_template and chart_template == 'in':
+                if fiscal_country.id != self.env.ref('base.in').id:
```

Comment:
```suggestion
            if move.company_id.chart_template != 'in' and move.country_code == "IN":
```
Usually moves are filtered based on `account_fiscal_country_id`